internal/cri/store/image/image.go

```diff
@@ -38,11 +38,25 @@ import (
 	imagespec "github.com/opencontainers/image-spec/specs-go/v1"
 )
 
+type RefKey struct {
+	// Ref of the image
+	Ref string
+	// Runtimehandler used for pulling this image
+	Platform string
```

Comment:
The comment mentions RuntimeHandler, but it seems we're using platform everywhere.
Shouldn't this actually be RuntimeHandler and we should ensure that the RuntimeHandler is actually used?

The same comment goes for the ImageIDKey struct and how it's used.

Reply:
No we moved away from the design of storing the runtimeHandler as it was making changes to core containerd packages which we did not want to do ahead with. This comment needs to be updated to "Platform used for pulling this image". Will address this and push new changes.